CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.226.0
+
+### Fixes
+
+* Upgrades passport to the latest version in order to fix security vulnerability. For passport methods that are internally used by Apostrophe everything is still working. For projects that are accessing the passport instance directly through `self.apos.login.passport`, some verifications may be necessary to avoid any comptibility issue. The internally used methods are `authenticate`, `use`, `serializeUser`, `deserializeUser`, `initialize`, `session`.
```

Comment:
Let's put this under a "Security" heading, not just "Fixes".

However, let's also reword it:

"... latest version in order to ensure session regeneration when logging in or out. This adds additional security to logins by mitigating any risks due to XSS attacks. Apostrophe is already robust against XSS attacks."

I don't think there is a plausible scenario to exploit the issue on a typical Apostrophe site, but more security is good.

Reply:
👍🏼